constants.py

```diff
@@ -642,9 +660,27 @@
 
 kHoodGearRatio = (1 / 5) * (13 / 360)
 
-kHoodMappingFunction = (
-    lambda x: 8.65 - 1.04 * x + 0.268 * x * x
-)  # see above for derivation method
+kHoodMappingValues = [
+    (1.6, 4),
+    (1.72, 9.5),
+    (1.89, 7.05),
+    (2.18, 8),
+    (2.41, 8.4),
+    (2.63, 7.425),
+    (2.76, 7),
+    (2.78, 9),
+    (2.84, 8.65),
+    (3.28, 7),
+    (3.5, 8.1),
+    (3.74, 8.7),
+    (3.78, 9.75),
+    (3.79, 9),
+    (4.2, 8.1),
+    (4.23, 9),
+    (4.51, 9.393333333),
+    (4.7, 10),
+    (5.49, 11),
+]  # see above for derivation method
```

Comment:
Better to duplicate the comment (if the other block is removed in another commit it will be easy to miss that this section needs updating)